feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync

```diff
@@ -40,17 +40,20 @@ start_service() {
     echo "Checking Redirector"
     redirector=$(uci get system.tip.redirector)
     if [ -z "$redirector" ]; then
-        [[ -f /usr/opensync/certs/redirector.txt ]] && redirector=$(cat /usr/opensync/certs/redirector.txt | tr -d '\r\n')
-        if [ -z "$redirector" ]; then
-            logger -t opensync "Contacting DigiCert for redirector address"
-            wlan_ap_redirector.sh
-        else
-            logger -t opensync "Restoring redirector ${redirector} after factory reset"
-            wlan_ap_redirector.sh ${redirector}
+        [[ -f /usr/opensync/certs/redirector.txt ]] && saved_redirector=$(cat /usr/opensync/certs/redirector.txt | tr -d '\r\n')
+        logger -t opensync "Contacting DigiCert for redirector address"
+        wlan_ap_redirector.sh
+        new_redirector=$(uci get system.tip.redirector)
+        if [ -z "$new_redirector" && -n "$saved_redirector" ]; then
+            logger -t opensync "No response from DigiCert, using saved redirector address ${saved_redirector}"
+            wlan_ap_redirector.sh ${saved_redirector}
+        else 
+            logger -t opensync "DigiCert returned redirector address ${new_redirector}"
```

Comment:
Where do we apply the new_redirector?

Reply:
We don't, it was already discovered and applied by the call to wlan_ap_redirector.sh at line 45.  If it hadn't been discovered and applied we'd use the saved redirector at line 49.